specs/__snapshots__/linkSpec.js.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Links Should render links with custom styles 1`] = `"%cLorem Ipsum  - www.google.com font-family:Times New Roman;font-size:16px;color:red;margin:12px 0px;text-decoration:underline;href:www.google.com;"`;
+exports[`Links Should render links with custom styles 1`] = `"%cLorem Ipsum  - www.google.com font-family:Times New Roman;font-size:16px;color:blue;margin:12px 0px;text-decoration:underline;href:www.google.com;"`;
```

Comment:
Same here. It seems the links are not being rendered with the custom styles passed, but with the defaults one.

Reply:
Found the cause. Please read following comment